tests/xgboost/test_xgboost_model_export.py

```diff
@@ -62,6 +62,7 @@ def test_model_save_load(xgb_model, model_path):
     mlflow.xgboost.save_model(xgb_model=model, path=model_path)
     reloaded_model = mlflow.xgboost.load_model(model_uri=model_path)
     reloaded_pyfunc = pyfunc.load_pyfunc(model_uri=model_path)
+    print(reloaded_pyfunc)
```

Comment:
I think it's a bit weird to add `print` here because this is not a xgboost-specific issue. Can we a new test (say `test_repr_can_be_called_withtou_run_id_or_artifact_path`) in `tests/pyfunc/test_model_export_with_class_and_artifacts.py` instead ?

Reply:
Right... I have to say that I am not very experienced with python (yet) 😄. I now tried to create a minimal, self-contained test following design patterns used in the test file you mentioned. Also, I saw elsewhere in this repo that `str(x)` was used to to test the print method, so I opted for this. Let me know what you think. 0fb0bf4 is the proof that the test is testing what I wanted to fix.